libs/langgraph/src/graph/graph.ts

```diff
@@ -463,7 +467,18 @@ export class Graph<
     // validate targets
     for (const node of Object.keys(this.nodes)) {
       if (!allTargets.has(node)) {
-        throw new Error(`Node \`${node}\` is not reachable`);
+        throw new UnreachableNodeError(
+          [
+            `Node \`${node}\` is not reachable.`,
+            "",
+            "If you are returning Command objects from your node,",
+            'make sure you are passing names of potential destination nodes as an "ends" array',
+            'into ".addNode(..., { ends: ["node1", "node2"] })".',
```

Comment:
quote formatting is weird here (some are `"`, and others are `'`). how is this passing the formatter?

Reply:
I have double quotes in the string so you need to use a different one